packages/rrweb/src/replay/index.ts

```diff
@@ -1703,18 +1687,22 @@ export class Replayer {
       appendNode(mutation);
     });
 
-    const startTime = Date.now();
+    const startTime = performance.now();
+
     while (queue.length) {
       // transform queue to resolve tree
       const resolveTrees = queueToResolveTrees(queue);
+
       queue.length = 0;
-      if (Date.now() - startTime > 500) {
+
+      if (performance.now() - startTime > 500) {
```

Comment:
```suggestion
      if (performance.now() - startTime > 150) {
```

500ms is an age... and this code is otherwise very fast